frontends/web/src/routes/account/summary/accountssummary.tsx

```diff
@@ -60,9 +61,23 @@ export function AccountsSummary({ accounts, devices }: TProps) {
   const [accountsTotalBalance, setAccountsTotalBalance] = useState<accountApi.TAccountsTotalBalance>();
   const [coinsTotalBalance, setCoinsTotalBalance] = useState<accountApi.TCoinsTotalBalance>();
   const [balances, setBalances] = useState<Balances>();
+  const { lightningConfig } = useLightning();
 
   const hasCard = useSDCard(devices);
 
+  // lightning account exists but is not from any connected or remembered keystores
+  const hasLightningFromOtherKeystore = (
+    lightningConfig.accounts.length !== 0
+    && (
+      accountsByKeystore.length === 0
+      || accountsByKeystore.some(({ keystore }) => {
+        return keystore.rootFingerprint !== lightningConfig.accounts[0].rootFingerprint;
+      })
+    )
+  );
+
+  const showTotalCoins = accountsByKeystore.length > 1 || (hasLightningFromOtherKeystore && accountsByKeystore.length === 1);
```

Comment:
Nit: what about doing smth like

`let keystores = accountsByKeystore.length;
if (hasLightningFromOtherKeystore) {
    keystores += 1;
}`
  
  and check for keystores > 1 below?
  
  I feel like it would be more readable

Reply:
I updated this to use `keystores`